client/src/components/chat/chat-item.tsx

```diff
@@ -53,14 +53,22 @@ export interface ChatItemData extends ChatMsg {
   name: string;
   color: string;
 }
+
+interface ChatProps {
+  displayMentorNames?: boolean;
+  heightSomething?: number;
```

Comment:
I was using `heightSomething` just as an example prop you might need :) I didn't mean literally use that name...it looks like no `height` or `width` props are needed?